package.json

```diff
@@ -30,6 +30,13 @@
     "test:validate": "./scripts/tests-validate.sh",
     "test:browser:filter": "vitest --run --coverage false --config vitest.browser.config.mts",
     "test:e2e": "vitest --run --config vitest.node.config.mts $(scripts/tests-find.sh --e2e)",
+    "bench:node": "vitest bench --run --config vitest.node.config.mts $(scripts/tests-find.sh)",
+    "bench:node:filter": "vitest bench --run --config vitest.node.config.mts",
+    "clinic:flame": "clinic flame --autocannon [ / ] -- node packages/${npm_config_package_name}/dist/profile.js",
```

Comment:
I've decided to still include these performance analysis scripts for digging into issues but they do require setting up a package with profiling infrastructure. I am open to removing them if we determine this is not germane to the PR

Reply:
Could we have these script within the `@internal/benchmarks` packages?

Some documentation on their usages might be beneficial as well.

Reply:
@petertonysmith94 I can definitely add some documentation on their usage although they are more for our use cases than consumers. 

I wouldn't put the `bench` commands in the `@internal/benchmarks` though since they rely on vitest config which is at the root, I think it would be awkward for a package to rely on files outside of it.

Nor would I put profiling in the `@internal/benchmarks` package because the intention is to use them with a specific package in the monorepo e.g.

```
npm_config_package_name=account pnpm clinic:flame  // runs flame against the account package
```

so it doesn't really have a use case outside of the monorepo

Reply:
@maschad Understood!

~Documentation would be good (doesn't have to be part of this PR) for our internal use would be wonderful.~